ss14432-at4524/project.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Scheduler for fog computing"
+author:
+  - Sripranav Suresh Kumar <ss14432@nyu.edu>
+  - Abitha Thankaraj<at4524@nyu.edu>
+---
+
+For our Big Data and Machine Learning final project we plan to develop a
+scheduler for fog/ edge computing.
+# Motivation
+
+Fog computing, also called Edge Computing, is intended for distributed computing where numerous 
+"peripheral" devices connect to a cloud. With the explosion of IoT devices and widespread availability 
+ of internet in the past decade, there has been a push to move computation to devices on the edge. While cloud computing
+ offers several advantages wih respect to elasticity and scalability, the distance between the edge devices and
+ the cloud gives rise to latency issues. This becomes a bottleneck in IoT applications
+ where results to computations are necessary in real-time (like in the case of disaster management
+and content delivery applications.) Service Level Agreements (SLAs) may also impose processing at 
+locations where the cloud provider does not have data centers. Latency and other limitations have 
+been discussed in this [paper](https://www.researchgate.net/profile/Rodolfo-Milito/publication/235409978_Fog_Computing_and_its_Role_in_the_Internet_of_Things/links/0deec531f19946228c000000/Fog-Computing-and-its-Role-in-the-Internet-of-Things.pdf).
+
+Resource management and task scheduling in Fog Computing have garnered interest especially since there has been a
```

Comment:
What key differences do you see between cloud computing and fog computing in terms of task scheduling?